.github/ISSUE_TEMPLATE/improve-existing-docs.yml

```
@@ -0,0 +1,64 @@
+name: Improve existing docs
+about: Make a suggestion to improve our existing documentation.
+title: ''
+labels: "content"
```

We shouldn't need to quote something like this:

```suggestion
labels: content
```